ovos_core/intent_services/__init__.py

```diff
@@ -199,6 +223,38 @@ def disambiguate_lang(message):
 
         return default_lang
 
+    def get_pipeline(self, skip_converse=False, skip_fallback=False):
+        # Create matchers
+        # TODO - from plugins
```

Comment:
will solve this TODO in https://github.com/OpenVoiceOS/ovos-core/pull/310

for now just want to make it configurable to clean up padatious/padacioso and allow more control for users